koku/cost_models/serializers.py

```diff
@@ -294,10 +310,10 @@ class Meta:
     def metric_map(self):
         """Map metrics and display names."""
         metric_map_by_source = defaultdict(dict)
-        metric_map = CostModelMetricsMap.objects.all()
+        metric_map = get_json(os.path.join(BASE_DIR, "api/metrics/data/cost_models_metric_map.json"))
```

Comment:
You've already got a constant defined in another file for this location, I'd use that.

Reply:
updated